### PR TITLE
New Portuguese translations

### PR DESCRIPTION
**Pull request by JANogueira**
New Portuguese translations for the following lexicon keys:
pt
pt-br

The files also include default English lexicon fallback in case of missing defaults or new additions.
PT-BR might require some tune-up due to minor semantic differences from Portuguese from Portugal.

Related with #168

### CODE REVIEW

File: core/components/clientconfig/lexicon/pt-br/default.inc.php [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+ * ClientConfig; if not, write to the Free Software Foundation, Inc., 59 Temple
+ * Place, Suite 330, Boston, MA 02111-1307 USA
+ *
+ * @package clientconfig
+ * @subpackage lexicon
+ * @language pt (Translated by João Nogueira - JANogueira)
+ * 
+ */
+
+include_once(dirname(dirname(__FILE__)).'/en/default.inc.php'); // fallback for missing defaults or new additions
```
Comment (Mark-H) [inline core/components/clientconfig/lexicon/pt-br/default.inc.php]: Please remove this - it is not necessary. MODX automatically falls back to English lexicons when they're not found in the loaded language.

File: core/components/clientconfig/lexicon/pt/default.inc.php [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+ * ClientConfig; if not, write to the Free Software Foundation, Inc., 59 Temple
+ * Place, Suite 330, Boston, MA 02111-1307 USA
+ *
+ * @package clientconfig
+ * @subpackage lexicon
+ * @language pt (Translated by João Nogueira - JANogueira)
+ * 
+ */
+
+include_once(dirname(dirname(__FILE__)).'/en/default.inc.php'); // fallback for missing defaults or new additions
```
Comment (Mark-H) [inline core/components/clientconfig/lexicon/pt/default.inc.php]: Same here.

File: core/components/clientconfig/lexicon/pt-br/default.inc.php (hunk 1 above)
Comment (JANogueira) [reply core/components/clientconfig/lexicon/pt-br/default.inc.php]: Done

File: core/components/clientconfig/lexicon/pt/default.inc.php (hunk 2 above)
Comment (JANogueira) [reply core/components/clientconfig/lexicon/pt/default.inc.php]: Done

**Comment by Mark-H** [comment]
Muito obrigado! :)

**Comment by JANogueira** [comment]
You are very welcome!
Cheers

**Pull request merged**